### PR TITLE
fix(travis): install auth, oauth and profile servers for travis

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

**Comment by zaach** [comment]
Looks like we do need the verifier:
```
fxa-oauth-server.assertion.ERROR: verify.error Error: connect ECONNREFUSED

    at exports._errnoException (util.js:746:11)

    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1010:19)

fxa-oauth-server.summary.ERROR: summary {"code":500,"errno":999,"method":"post","path":"/v1/authorization","agent":"Mozilla/5.0 (X11; Linux x86_64; rv:38.0) Gecko/20100101 Firefox/38.0","t":25,"client_id":"98e6508e88680e1a","auth":null,"payload":["client_id","scope","assertion","response_type","ttl","access_type","redirect_uri"],"stack":"Error: connect ECONNREFUSED\n    at exports._errnoException (util.js:746:11)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1010:19)"}
```

File: tests/ci/deps.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+npm i
+# issue https://github.com/mozilla/fxa-profile-server/issues/138
+npm i rimraf@^2.2.8
+NODE_ENV=dev npm start &
+cd ../..
+
+# Verifier
+
+git clone https://github.com/vladikoff/browserid-verifier.git -b http
+cd browserid-verifier; npm i;
```
Comment (pdehaan) [inline tests/ci/deps.sh]: Should this just be `$ npm i vladikoff/browserid-verifier#http`

Comment (vladikoff) [inline tests/ci/deps.sh]: Sure, for consistency we can do that i think

**Comment by zaach** [comment]
YEEEAHH

**Pull request merged**